internal/msg/errormsg.go

```diff
@@ -58,6 +58,8 @@ const (
 	MissingRegion = "no sauce region set"
 	// EmptySuite indicates no suites in the config
 	EmptySuite = "no suites defined"
+	// EmptySuiteWithSharding indicates no suites in the config after sharding processing
+	EmptySuiteWithSharding = "sharding configuration resulted in no suites defined"
```

Comment:
Hm, I'd be confused by the message, since suites are defined from a user's perspective. It's rather that the suite's configuration resulted in no tests found. Maybe "sharding configuration resulted in no matching tests"?

Reply:
Sounds better ! Was not 100% convinced on the phrasing but that was the best I had 😄